### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by bluetimer**
added travis declaration 
Closes https://github.com/manu-chroma/username-availability-checker/issues/38

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: python
 python:
-  - 3.6
+- "3.4"
+- "3.5"
+- "3.6"
```
Comment (andrewda) [inline .travis.yml]: Indent these lines with 2 spaces. Also quotes aren't necessary.

**Comment by andrewda** [comment]
Also, in the future, please try to keep working on the same PR instead of closing it and creating a new one https://github.com/manu-chroma/username-availability-checker/pull/119

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 language: python
 python:
+  - 3.4                                            
+  - 3.5  
   - 3.6
-
+
```
Comment (yukiisbored) [inline .travis.yml]: Remove this unwanted whitespaces. Try keep your amount of changes as small as you can.

**Comment by yukiisbored** [comment]
Also, squash your commits since this is just one change. Feel free to ask us on how to do it in the Zulip/Gitter chat.

**Comment by yukiisbored** [comment]
Your commit message does not follow our commit guidelines. See https://coala.io/commit.

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,9 @@
 language: python
 python:
+  - 3.4
```
Comment (andrewda) [inline .travis.yml]: Also remove trailing whitespace on this line and on the line below

**Comment by bluetimer** [comment]
.travis.yml:added python versions in travis declaration  
added python 3.4 3.5 3.6 version in travis 
closes https://github.com/manu-chroma/username-availability-checker/issues/38

File: README.md [hunk 4]
```diff
@@ -48,6 +48,8 @@ To use, go to address generated by flask app in a browser.
 
 ### Running tests for flask backend
 
+Runs on python 3.4,3.5
+
 Tested on python 3.6
```
Comment (andrewda) [inline README.md]: Maybe instead of the addition above, something like,

"Tested on Python 3.4, 3.5 and 3.6"

**Pull request merged**